pipelines/utilities/longhorn_manifest.sh

```
@@ -56,18 +69,47 @@ generate_longhorn_yaml_manifest() {
+  yq -i 'select(.kind == "ConfigMap" and .metadata.name == "longhorn-default-setting").data."default-setting.yaml"="registry-secret: docker-registry-secret"' "${LONGHORN_MANIFEST_PATH}"
   # (2) modify images to point to custom registry
   if [[ ! -z "${REGISTRY_URL}" ]]; then
-    sed -i "s/longhornio\//${REGISTRY_URL}\/longhornio\//g" "/tmp/longhorn.yaml"
+    sed -i "s/longhornio\//${REGISTRY_URL}\/longhornio\//g" "${LONGHORN_MANIFEST_PATH}"
   fi
 }
 
-install_longhorn_by_manifest(){
-  kubectl apply -f "/tmp/longhorn.yaml"
+install_longhorn(){
+  LONGHORN_NAMESPACE="longhorn-system"
```

Shellcheck (0.10.0)

[warning] LONGHORN_NAMESPACE appears unused. Verify use (or export if used externally).

(SC2034)